tests/console/rpm.pm

```diff
@@ -69,7 +69,7 @@ sub run {
         # Get detailed information for a package
         assert_script_run("rpm -qi $installed_pkg");
         # Read changelog of a package
-        assert_script_run("rpm -q --changelog $installed_pkg", 90);
+        assert_script_run("rpm -q --changelog $installed_pkg", 180);
```

Comment:
I the problem is that it taking too long to pass log over a network . so 180 will help until there will be bigger network problem so it will timeout even with 180 :) what if instead we will do something like this ? : 
```suggestion
        assert_script_run("rpm -q --changelog $installed_pkg | tail -n 40", 90);
```
I don't think it will harm testing coverage

Reply:
This is good idea! VR running.